docs/Configuration/fleet-server-configuration.md

```diff
@@ -2896,6 +2896,38 @@ The duration between DEP device syncing (fetching and setting of DEP profiles).
   mdm:
     apple_dep_sync_periodicity: 10m
   ```
+##### mdm.windows_wstep_identity_cert_bytes
+> Windows MDM features are not ready for production and are currently in development. These features are disabled by default.
+
+The content of the Windows WSTEP identity certificate. An X.509 certificate, PEM-encoded.
+- Default value: ""
+- Environment variable: fleet_mdm_windows_wstep_identity_cert_bytes  
```

Comment:
```suggestion
- Environment variable: fleet_mdm_windows_wstep_identity_cert_bytes  
```

Ah, shoot. I missed this on my first review.

In Fleet, and in most products, environment variables are in all caps. So, we want the env variable in both config options to be in all caps. 

See this config options as an example: https://fleetdm.com/docs/configuration/fleet-server-configuration#mdm-apple-apns-key-bytes